cloverage/src/cloverage/coverage.clj

```diff
@@ -155,8 +155,14 @@
         (println "Failing build as coverage is below threshold of" failure-threshold "%"))
       failed?)))
 
+(defn launch-custom-report
```

Comment:
from the tests `report-sym` would be `cloverage.custom-reporter/reporter-fn` so it's resolved then called.  So long as you have it on the classpath it will be ok.

Reply:
I swear I looked for this function and didn't see it in the old diff. My bad :)